test/controllers/v1/permission_grant_controller_test.exs

```diff
@@ -2,6 +2,12 @@ defmodule Cog.V1.PermissionGrantController.Test do
   use Cog.ModelCase
   use Cog.ConnCase
 
+  # When these tests were written, permissions were allowed to be
+  # associated with users, groups, or roles. This model has been
+  # simplified to only allow permissions to be associated with roles,
+  # but the parameterized test scaffolding has been retained for
+  # the time being. More information about this structure is below.
+  #
```

Comment:
Are we leaving this in because we're going to add something soon that can use it again, or just for expediency? If the latter, can we put an issue on the backlog to remove it?